libs/fabric/src/lib/components/marquee-selection/marquee-selection.component.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  Input,
+  NgZone,
+  Renderer2,
+  ViewChild,
+} from '@angular/core';
+import { ReactWrapperComponent } from '@angular-react/core';
+import { IMarqueeSelectionProps } from 'office-ui-fabric-react/lib/MarqueeSelection';
+
+@Component({
+  selector: 'fab-marquee-selection',
+  exportAs: 'fabMarqueeSelection',
+  template: `
+    <MarqueeSelection
+      #reactNode
+      [selection]="selection"
+      [className]="className"
+      [componentRef]="componentRef"
+      [isDraggingConstrainedToRoot]="isDraggingConstrainedToRoot"
+      [isEnabled]="isEnabled"
+      [rootProps]="rootProps"
+      [styles]="styles"
+      [theme]="theme"
+      [ShouldStartSelection]="onShouldStartSelection">
+      <ReactContent><ng-content></ng-content></ReactContent>
+    </MarqueeSelection>
+  `,
+  styles: ['react-renderer'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FabMarqueeSelectionComponent extends ReactWrapperComponent<IMarqueeSelectionProps> {
+  
+  @ViewChild('reactNode')
+  protected reactNodeRef: ElementRef;
+
+  // Required members
+  @Input()
+  selection: IMarqueeSelectionProps['selection'];
+
+  // Optional members
+  @Input()
+  className?: IMarqueeSelectionProps['className'];
+  @Input()
+  componentRef?: IMarqueeSelectionProps['componentRef'];
+  @Input()
+  isDraggingConstrainedToRoot?: IMarqueeSelectionProps['isDraggingConstrainedToRoot'];
+  @Input()
+  isEnabled?: IMarqueeSelectionProps['isEnabled'];
+  @Input()
+  onShouldStartSelection?: (ev: MouseEvent) => boolean;
```

Comment:
It was working locally though [onShouldStartSelection]="callback" though it does seem a bit awkward to have a property prefixed with "on" between square brackets instead of parentheses. I'll try it in the AOT build as well

Reply:
![image](https://user-images.githubusercontent.com/260431/48488296-0fbabe80-e829-11e8-8c59-7c72a104bb3c.png)

From the demo app.

Reply:
Verified "build:demo:prod" and "build:docs:prod" are both working